multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/UpdateServiceParametersStep.java

```diff
@@ -27,9 +31,12 @@ protected OperationExecutionState executeOperation(ProcessContext context, Cloud
         }
         getStepLogger().info(Messages.UPDATING_SERVICE, service.getName());
 
-        client.updateServiceParameters(service.getName(), service.getCredentials());
-
-        getStepLogger().debug(Messages.SERVICE_UPDATED, service.getName());
+        try {
+            client.updateServiceParameters(service.getName(), service.getCredentials());
+            getStepLogger().debug(Messages.SERVICE_UPDATED, service.getName());
+        } catch (CloudOperationException e) {
+            processServiceUpdateFailure(context, service, e, PARAMETERS);
+        }
 
         return OperationExecutionState.EXECUTING;
```

Comment:
In case of error, you do not need to monitor the service again
return OperationExecutionState.FINISHED instead of EXECUTING